db/app.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const {
+  getHealthcheck,
+  getTopics,
+} = require("./controllers/topicsController.js");
+const { handleCustomErrors } = require("./error");
+
+const app = express();
+
+app.use(express.json());
+
+app.get("/api/healthcheck", getHealthcheck);
```

Comment:
Im not sure what this healthcheck thing is. Please try only add the code that the ticket is asking for.